src/definition/def_types.rs

```diff
@@ -408,8 +407,9 @@ impl GenericDec {
             byte_idx: 0,
             url: url.clone(),
             type_str: String::new(),
-            completion_kind: CompletionItemKind::Variable,
-            symbol_kind: SymbolKind::Unknown,
+            completion_kind: CompletionItemKind::VARIABLE,
+            // FIXME: check if this replacement is correct
+            symbol_kind: SymbolKind::NULL,
```

Comment:
I'm not sure why that was unknown previously, I would assume NULL or VARIABLE are fine as replacements